.github/workflows/docker.yml

```diff
@@ -31,18 +33,26 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Compute labels
+        id: labels
+        run: |
+          REF=$(echo "${{ github.ref_name }}" | sed 's/\//_/g')
+          echo "ref=$REF" >> $GITHUB_OUTPUT
+          DATE=$(echo "${{ github.event.repository.updated_at }}" | head -c 10)
+          echo "date=$DATE" >> $GITHUB_OUTPUT
+          echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
```

Comment:
It would be nice to add some comments here, e.g., the expected format for each variable.

Reply:
Added some explanatory comments.

Reply:
That's a pretty decent piece of documentation, thanks.